build-binutils.py

```diff
@@ -155,17 +158,20 @@ def invoke_configure(build_folder, install_folder, root_folder, target,
             'CXXFLAGS=-O2 -march=%s -mtune=%s' % (host_arch, host_arch)
         ]
     else:
-        configure += [
-            'CFLAGS=-O2',
-            'CXXFLAGS=-O2'
-        ]
+        configure += ['CFLAGS=-O2', 'CXXFLAGS=-O2']
```

Comment:
unrelated: Has anybody seen any benefit to experimenting with O3 here?

Reply:
No, just build time is a bit longer.
I noticed this while building on Drone CI, over 48 cores. 
With O3, it exceeded time limit of 1 hr. While with O2, it built successfully in 57 minutes.